src/commands/asset-compute/index.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2019 Adobe. All rights reserved.
```

Comment:
I think we're in 2020 :D.

Reply:
As mentioned before, it doesn't really matter, there will be a mix in the codebase anyway.